src/titles/titleFormatterData.ts

```diff
@@ -274,4 +388,4 @@ export const notStartOfSentence = new Set([
,"T"],["𝚻","T"],["𝛵","T"],["𝕋","T"],["𝕿","T"],["𝑻","T"],["𐊱","T"],["𐊗","T"],["𝖳","T"],["𝙏","T"],["🝨","T"],["𝝩","T"],["𝞣","T"],["𝚃","T"],["𝘛","T"],["𝑇","T"],["ꓔ","T"],["⟙","T"],["𝐓","T"],["Ⲧ","T"],["𝗧","T"],["⊤","T"],["𝔗","T"],["𝒯","T"],["🅣","T"],["⏉","T"],["Ⓣ","T"],["Ｔ","T"],["ᴛ","T"],["ᵀ","T"],["𝓣","T"],["ⓣ","t"],["ｔ","t"],["ᵗ","t"],["⒯","t"],["𝐭","t"],["𝑡","t"],["𝒕","t"],["𝔱","t"],["𝕥","t"],["𝖙","t"],["𝗍","t"],["𝘁","t"],["𝘵","t"],["𝙩","t"],["𝚝","t"],["𝓉","t"],["𝓽","t"],["🅄","U"],["🄤","U"],["🆄","U"],["🅤","U"],["Ⓤ","U"],["ᵁ","U"],["ᵘ","U"],["⒰","U"],["𝐔","U"],["𝑈","U"],["𝑼","U"],["𝒰","U"],["𝓤","U"],["𝔘","U"],["𝕌","U"],["𝖀","U"],["𝖴","U"],["𝗨","U"],["𝘜","U"],["𝙐","U"],["𝚄","U"],["𝘂","u"],["𝘶","u"],["𝙪","u"],["𝚞","u"],["𐓶","u"],["ⓤ","u"],["ｕ","u"],["ᵤ","u"],["𝐮","u"],["𝑢","u"],["𝒖","u"],["𝓊","u"],["𝓾","u"],["𝔲","u"],["𝕦","u"],["𝖚","u"],["𝗎","u"],["𝑉","V"],["𝒱","V"],["𝕍","V"],["𝗩","V"],["🄥","V"],["🅅","V"],["ꓦ","V"],["𝑽","V"],["𝖵","V"],["𝘝","V"],["𝚅","V"],["𝙑","V"],["𝐕","V"],["🆅","V"],["🅥","V"],["Ⓥ","V"],["ᵛ","V"],["⒱","V"],["Ⅴ","V"],["ⴸ","V"],["𝓥","V"],["∨","v"],["ⅴ","v"],["𝐯","v"],["𝑣","v"],["𝒗","v"],["𝕧","v"],["𝗏","v"],["ⓥ","v"],["ｖ","v"],["ᴠ","v"],["ᵥ","v"],["𝙫","v"],["𝚟","v"],["𝛎","v"],["𝜈","v"],["𝝂","v"],["𝝼","v"],["𝞶","v"],["𝘷","v"],["𝘃","v"],["𝓿","v"],["𝓋","v"],["𝐖","W"],["𝑊","W"],["𝓦","W"],["𝕎","W"],["𝖶","W"],["𝗪","W"],["𝙒","W"],["𝚆","W"],["🄦","W"],["🅆","W"],["𝑾","W"],["ꓪ","W"],["𝒲","W"],["𝘞","W"],["🆆","W"],["Ⓦ","W"],["🅦","W"],["ｗ","W"],["ᵂ","W"],["ʷ","W"],["⒲","W"],["𝐰","w"],["𝑤","w"],["𝒘","w"],["𝔀","w"],["𝔴","w"],["𝕨","w"],["𝖜","w"],["𝗐","w"],["𝘄","w"],["𝘸","w"],["𝙬","w"],["𝚠","w"],["ⓦ","w"],["𝓌","w"],["𝛘","X"],["𝜒","X"],["𝝌","X"],["𝞆","X"],["𝟀","X"],["𝑿","X"],["𝛸","X"],["🄧","X"],["🅇","X"],["𐌗","X"],["Ⲭ","X"],["ꓫ","X"],["𝞦","X"],["𝘟","X"],["𐊐","X"],["𝚾","X"],["𝝬","X"],["𝜲","X"],["𐌢","X"],["𝖷","X"],["𝑋","X"],["𝕏","X"],["𐊴","X"],["𝗫","X"],["🆇","X"],["🅧","X"],["Ⓧ","X"],["𝓧","X"],["Ｘ","X"],["𝒳","X"],["ˣ","X"],["⒳","X"],["Ⅹ","X"],["╳","X"],["ⅹ","X"],["ⵝ","X"],["𝙓","X"],["𝚇","X"],["𝐗","X"],["ⅹ","x"],["𝑥","x"],["𝒙","x"],["𝓍","x"],["𝗑","x"],["𝘅","x"],["ⓧ","x"],["ｘ","x"],["×","x"],["ₓ","x"],["𝘹","x"],["𝐱","x"],["𝚡","x"],["𝔁","x"],["𝙭","x"],["𝒴","Y"],["🄨","Y"],["𝓨","Y"],["𝖸","Y"],["𝘠","Y"],["𝙔","Y"],["𝚼","Y"],["𝛶","Y"],["𝝪","Y"],["𝞤","Y"],["𝚈","Y"],["𝑌","Y"],["𝗬","Y"],["𝐘","Y"],["𝒀","Y"],["𝜰","Y"],["𐊲","Y"],["🆈","Y"],["🅨","Y"],["Ⓨ","Y"],["𝐲","y"],["𝑦","y"],["𝒚","y"],["𝓎","y"],["𝔂","y"],["𝗒","y"],["𝘆","y"],["𝘺","y"],["𝙮","y"],["𝚢","y"],["ʏ","y"],["ỿ","y"],["ꭚ","y"],["ℽ","y"],["𝛄","y"],["𝛾","y"],["𝜸","y"],["𝝲","y"],["𝞬","y"],["🅈","y"],["ⓨ","y"],["ｙ","y"],["ƴ","y"],["ʸ","y"],["⒴","y"],["🄩","Z"],["🅉","Z"],["ꓜ","Z"],["𝗭","Z"],["𝐙","Z"],["𝘡","Z"],["🆉","Z"],["🅩","Z"],["Ⓩ","Z"],["Ｚ","Z"],["ᶻ","Z"],["⒵","Z"],["ℤ","Z"],["𝑍","z"],["𝒁","z"],["𝒵","z"],["𝓩","z"],["𝙕","z"],["𝚉","z"],["𝚭","z"],["𝛧","z"],["𝜡","z"],["𝝛","z"],["𝞕","z"],["𝐳","z"],["𝑧","z"],["𝒛","z"],["𝓏","z"],["𝗓","z"],["𝘇","z"],["𝘻","z"],["𝙯","z"],["𝚣","z"],["ⓩ","z"],["ｚ","z"],["ᴢ","z"],["𝔃","z"]]`));
```

Comment:
What was changed here?

Reply:
I didn't change anything there. From my OP:

>Edit: I didn't change fancyTextConversions but for some reason in the commit it shows as being modified.

Reply:
newline at the EOF